### PR TITLE
Bayesian wilcoxon signed rank - flip the samples and use testValue

### PR DESCRIPTION
**Pull request by JohnnyDoorn**
Fix https://github.com/jasp-stats/jasp-issues/issues/1038
and use testvalue

### CODE REVIEW

**Comment by AlexanderLyNL** [comment]
All plots seem to fail, @vandenman how problematic is this? Is this related to the jaspGraphs upgrade?

File: JASP-Engine/JASP/R/ttestbayesianonesample.R [hunk 1]
```diff
@@ -121,7 +121,7 @@ TTestBayesianOneSample <- function(jaspResults, dataset, options, state = NULL)
           
           .setSeedJASP(options)
           r <- try(.signRankGibbsSampler(
```
Comment (AlexanderLyNL) [inline JASP-Engine/JASP/R/ttestbayesianonesample.R:123]: The name `r` is not so informative, perhaps use a more descriptive name

File: JASP-Engine/JASP/R/ttestbayesianonesample.R [hunk 2]
```diff
@@ -134,11 +134,11 @@ TTestBayesianOneSample <- function(jaspResults, dataset, options, state = NULL)
 
           } else {
             
-            ttestResults[["delta"]][[var]]  <- r[["deltaSamples"]] * -1
+            ttestResults[["delta"]][[var]]  <- r[["deltaSamples"]]
             ttestResults[["rHat"]][[var]]  <- r[["rHat"]]
             
             bf.raw <- .computeBayesFactorWilcoxon(
```
Comment (AlexanderLyNL) [inline JASP-Engine/JASP/R/ttestbayesianonesample.R:140]: bfRaw would be better

**Review by AlexanderLyNL — APPROVED** [review]
This seems fine. I added some style comments that should be looked into next year.

**Comment by vandenman** [comment]
> All plots seem to fail, @vandenman how problematic is this? Is this related to the jaspGraphs upgrade?

Yes, I kinda didn't consider that we want the unit tests to keep working here. Either way, if @JohnnyDoorn can also add a PR to jasp-stats/jaspTTest then we can see if the unit tests pass there.

**Comment by JohnnyDoorn** [comment]
@vandenman the changes are already in TTest, because I think I had accidentally checked out the upstream branch and pushed, so it was added without a PR - gonna make sure it doesn't happen in the future.

**Review by vandenman — APPROVED** [review]
changes are the same in jaspTTests and there the unit tests pass

**Pull request merged**